### ISSUE TITLE
Didn't find class "com.giphy.sdk.ui.r

### ISSUE THREAD

**Issue by HirogaKatageri** [Giphy/giphy-android-sdk-ui-example#44]
Create and tested a release apk, crashed at startup.

Giphy SDK version: 1.2.6

```
Caused by java.lang.ClassNotFoundException: Didn't find class "com.giphy.sdk.ui.r" on path: DexPathList[[zip file "/system/framework/android.test.mock.jar", zip file "/system/framework/android.test.runner.jar", zip file "/data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/base.apk", zip file "/data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/split_config.armeabi_v7a.apk", zip file "/data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/split_config.xxhdpi.apk"],nativeLibraryDirectories=[/data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/lib/arm, /data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/base.apk!/lib/armeabi-v7a, /data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/split_config.armeabi_v7a.apk!/lib/armeabi-v7a, /data/app/tv.nexusgaming.streaming-i0j0xGQt80eliRqsu-oI6g==/split_config.xxhdpi.apk!/lib/armeabi-v7a, /system/lib, /system/vendor/lib]]
       at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:125)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
       at com.giphy.sdk.ui.d.<init>(:30)
       at com.giphy.sdk.ui.c.<init>(:17)
       at com.giphy.sdk.ui.a.a(:45)
       at com.giphy.sdk.ui.l.a(:98)
       at com.giphy.sdk.ui.Giphy.configure(:81)
       at tv.nexusgaming.streaming.NexplayApplication.onCreate(NexplayApplication.kt:66)
       at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1127)
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6111)
       at android.app.ActivityThread.-wrap1()
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1801)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:7025)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

**Comment by PiotrPrus** [comment]
I had that problem. It is solved in 1.2.7

**Comment by HirogaKatageri** [comment]
Yeah, @PiotrPrus I updated it to 1.2.8 now and the issue is gone. We did update to 1.2.7 previously, which had the change on one of the listeners. Apparently, we had a mistake on our merges and accidentally downgraded the version. Thought we were still using the latest one.
